lib/autoload/filename_sanitizer.rb

```diff
@@ -0,0 +1,8 @@
+module FilenameSanitizer
```

Comment:
Currently there is no use case for path.  And I think I do need a method only for filename.

Suppose there is a string `a/bc.txt`, if it's a path, then it's valid. but if it's filename, it's not.